packages/api/src/store/rabbit-queue.ts

```diff
@@ -81,12 +90,18 @@ export default class MessageQueue {
     await this.channel.sendToQueue(QUEUE_NAME, msg);
   }
 
+  public async publish (route: string, msg: object): Promise<void> {
+    console.log(`publishing to ${route} : ${JSON.stringify(msg)}`);
+    await this.channel.publish(EXCHANGE_NAME, route, msg);
```

Comment:
I think we should also have a `{ persistent: true }` option on these events otherwise they will be lost on a rabbitmq server restart (even if the queues/exchanges themselves are `durable`). Both here and in `sendToQueue` above.